### PR TITLE
New pullquote block design

### PR DESCRIPTION
**Pull request by luminuu**
Referring to issue #183 about the new pullquote block designs.

### CODE REVIEW

**Comment by luminuu** [comment]
Updated the smart quote as Mel suggested, now it's ready for review!

File: assets/css/ie-editor.css [hunk 1]
```diff
@@ -1364,26 +1364,38 @@ p.has-background {
...
+	border-top-width: 3px;
+	border-bottom-color: #28303d;
+	border-bottom-style: solid;
+	border-bottom-width: 3px;
 	color: #28303d;
 }
 
+.wp-block-pullquote::before {
+	color: currentColor;
+	content: "“";
```
Comment (pattonwebz) [inline assets/css/ie-editor.css]: Generally unicode chars are used for CSS `content` if any non-standard chars are added there.
```suggestion
	content: "\201C";
```

Comment (luminuu) [reply assets/css/ie-editor.css]: Thank you! Changed that one.

**Review by melchoyce — CHANGES_REQUESTED** [review]
Let's decrease the padding on the solid style on narrower screens, right now the text gets pretty cramped:



Otherwise, looks good to me!

**Review by melchoyce — APPROVED** [review]
LGTM

**Comment by carolinan** [comment]
Do we want to limit the width of the left and right aligned pullquote to 50%? When the content is long enough, the quotes are no longer aligned but looks like they are normal width without alignments.

**Comment by carolinan** [comment]
In the editor, left and right aligned pull quotes needs spacing to the right and left of the block respectively.

It is less visible on the right:
pull quotes in the editor is missing spacing

**Comment by carolinan** [comment]
Related: https://github.com/WordPress/twentytwentyone/pull/250

**Comment by melchoyce** [comment]
> Do we want to limit the width of the left and right aligned pullquote to 50%? When the content is long enough, the quotes are no longer aligned but looks like they are normal width without alignments.

Let's explore this in a new issue, I'll want to try out some ideas.

**Comment by melchoyce** [comment]
@carolinan Moved your feedback to https://github.com/WordPress/twentytwentyone/issues/282

**Pull request merged**